SCRAM/hooks/runtime/00-nvidia-drivers

```diff
@@ -17,12 +43,13 @@ else
   fi
 fi
 
-if [ "$NVIDIA_VERSION" ]; then
-  # if a driver is available, assume that a library is available as well
-  true
-else
-  # otherwise, use the library packaged with CMSSW
-  if [ -e "${CUDA_BASE}/drivers" ] ; then
-    echo "RUNTIME:path:append:LD_LIBRARY_PATH=${CUDA_BASE}/drivers"
-  fi
+# check the version of libcuda.so bundled with CMSSW
+CMS_NVIDIA_VERSION=$(library_version $(readlink -f ${CUDA_BASE}/drivers/libcuda.so))
+
+if [ "$NVIDIA_VERSION" ] && (( $(parse_version $CMS_NVIDIA_VERSION) <= $(parse_version $NVIDIA_VERSION) )); then
```

Comment:
@fwyzard , I used `<=` instead of `<` here.

Reply:
:+1: